clients/gtest/matmul_gtest.yaml

```diff
@@ -881,6 +881,36 @@ Tests:
   unit_check: 1
   algo_method: [0, 2]
 
+- name: matmul_groupedgemm_bf16
+  category: pre_checkin
+  function:
+    matmul: *hpa_bf16_precision
+  M: [127, 129]
+  N: [127, 129]
+  K: [127, 129]
+  transA_transB: *transA_transB_range
+  grouped_gemm: [2]
+  alpha: 1
+  beta: [ 0.0, 2.0 ]
+  unit_check: 1
+  algo_method: [2]
+  gpu_arch: '94[0-2]'
```

Comment:
you didn't put 942 yaml 
here should be only '942'

Reply:
Thanks for your comment.
I changed to 942.